databases/week2/class_n_homeWork_1_3.sql

```diff
@@ -0,0 +1,250 @@
+insert into user_task (user_id, task_id) values(4, 20);
+insert into user_task (user_id, task_id) values(5, 20);
+
+insert into user_task (user_id, task_id) values(5, 15);
+insert into user_task (user_id, task_id) values(6, 10);
+insert into user_task (user_id, task_id) values(6, 7);
+insert into user_task (user_id, task_id) values(6, 27);
+insert into user_task (user_id, task_id) values(7, 33);
+insert into user_task (user_id, task_id) values(7, 18);
+insert into user_task (user_id, task_id) values(7, 23);
+insert into user_task (user_id, task_id) values(8, 26);
+insert into user_task (user_id, task_id) values(8, 30);
+insert into user_task (user_id, task_id) values(8, 11);
+insert into user_task (user_id, task_id) values(9, 34);
+insert into user_task (user_id, task_id) values(9, 15);
+insert into user_task (user_id, task_id) values(9, 1);
+insert into user_task (user_id, task_id) values(10, 29);
+insert into user_task (user_id, task_id) values(10, 16);
+insert into user_task (user_id, task_id) values(10, 1);
+insert into user_task (user_id, task_id) values(11, 26);
+insert into user_task (user_id, task_id) values(11, 27);
+insert into user_task (user_id, task_id) values(11, 17);
+insert into user_task (user_id, task_id) values(11, 2);
+insert into user_task (user_id, task_id) values(1, 3);
+insert into user_task (user_id, task_id) values(2, 6);
+insert into user_task (user_id, task_id) values(3, 8);
+insert into user_task (user_id, task_id) values(4, 9);
+insert into user_task (user_id, task_id) values(5, 12);
+insert into user_task (user_id, task_id) values(6, 14);
+insert into user_task (user_id, task_id) values(7, 21);
+insert into user_task (user_id, task_id) values(9, 28);
+insert into user_task (user_id, task_id) values(10, 31);
+insert into user_task (user_id, task_id) values(11, 32);
+
+
+--CLass Exercise
+--Get all the tasks assigned to Pavel
+select t.*, u.name
+from task as t
+join user_task as ut
+on t.id = ut.task_id
+join `user` as u
+on ut.user_id = u.id
+where u.name like "Pavel%"
+
+--Find how many tasks each user is responsible for;
+select user.name, count(task.id) as no_of_taskAssign
+from `user`
+join user_task on user.id = user_task.user_id
+JOIN task ON user_task.task_id= task.id
+group BY user.name
+
+--Find how many tasks with a status=Done each user is responsible for;
+select user.name, count(task.id) as no_of_taskComplete
+from `user`
+join user_task on user.id = user_task.user_id
+JOIN task ON user_task.task_id= task.id
+join status on task.status_id = status.id
+where status.name = "Done"
+group BY user.name
+
+
+--Home-Work Part-1
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (36, 'DO homeworks', 'Title says it all, just do your homework', '2018-12-17 03:48:18', '2018-12-27 16:48:59','2019-01-20 15:05:39' , 2, 1);
```

Comment:
As the id is set to autoincrement, you don't need to pass a value. You can just do `INSERT INTO task (title,...) VALUES ('Do homeworks',...)`

Reply:
Ok noted, And I also made the changes in the query accordingly.